iwyu_tool.py

```diff
@@ -120,18 +138,8 @@ def run_iwyu(dbentry, extra_args, verbose):
     if verbose:
         print('# ' + ' '.join(command))
 
-    # Environment dictionary handling from https://stackoverflow.com/a/4453495
-    # ensures that: The current environment is not altered, instead it is
-    # copied for the child process, then the directory of iwyu_tool.py is
-    # prepended to the PATH variable.
-    env = os.environ.copy()
-    path_variable = os.path.dirname(__file__)
-    if 'PATH' in env:
-        path_variable += os.pathsep + env['PATH']
-    env['PATH'] = path_variable
```

Comment:
Sure -- instead of relying on the `PATH` variable in the environment to help `subprocess` locate `include-what-you-use`, I just locate it up-front with `find_include_what_you_use()` and then use the absolute path.